elfo-macros-impl/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+extern crate proc_macro;
+
+pub mod message;
+pub mod msg;
```

Comment:
It's better to expose only `message_impl` and `msg_impl` without exposing the internal structure of modules, which can be changed in the future.